pkg/kubelet/container/testing/fake_runtime_helper.go

```diff
@@ -53,7 +53,7 @@ func (f *FakeRuntimeHelper) GetPodDNS(pod *v1.Pod) (*runtimeapi.DNSConfig, error
 		Options:  f.DNSOptions}, f.Err
 }
 
-// This is not used by docker runtime.
+// GeneratePodHostNameAndDomain is not used by docker runtime.
```

Comment:
please add a description before this note. Something like

```suggestion
// GeneratePodHostNameAndDomain on fake runtime helper returns fake hostname and domain. It is not used by docker runtime.
```

Reply:
Thanks for your review! 

I changed it and also added description for other methods.